tests/conftest.py

```diff
@@ -209,6 +210,33 @@ def temp_config(config_overrides: dict, merge_strategy="override"):
             yield get_config()
 
 
+def get_mutable_config_copy(base_config: Optional[aconfig.ImmutableConfig] = None):
```

Comment:
ooof :D

Reply:
yerp. This should really be added to `aconfig`